neuralprophet/plot_model_parameters_plotly.py

```diff
@@ -790,7 +792,7 @@ def plot_custom_season(m, comp_name, quantile, multiplicative=False, df_name="__
             fill="none",
         )
     )
-    padded_range = get_dynamic_axis_range(list(range(len(t_i))), type="numeric")
+    padded_range = get_dynamic_axis_range(t_i, type="numeric")
```

Comment:
is this code correct? Looks so much simpler than before (if yes, just resolve the comment)